packages/ses/test/test-console-error-trap.js

```diff
@@ -0,0 +1,24 @@
+import test from 'ava';
+import { exec } from 'child_process';
+import { dirname, join } from 'path';
+
+const cwd = join(
+  dirname(new URL(import.meta.url).pathname),
+  'console-error-trap',
+);
+
+test.cb('errors reveal their stacks', t => {
```

Comment:
What is `test.cb`? I don't remember.

Reply:
I don’t either, but it was used in the packaging test after it got ported to Ava so I’ll look it up to learn why. Probably ensures that the test ends at `t.end()` instead of the resolution of the returned promise.

Reply:
`test.cb` is as I intuited above. `test` expects the return value of the testback to signal that the test has ended either by returning `undefined` or a promise that resolves to `undefined`, whereas `test.cb` is more like `tap`—the test ends only with an explicit call to `t.end()`.